app/main.py

```diff
@@ -1,3 +1,16 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+from functools import wraps
+
+
+def cache(func: Callable) -> Callable:
+    result: dict = {}
+
+    @wraps(func)
+    def inner(*args) -> int:
+        if args not in result.keys():
+            result[args] = func(*args)
```

Comment:
add kwargs as well, please.

Reply:
I add kwargs, but I dont't know is it right